movo_robot/movo_bringup/launch/platform/movo.launch

```diff
@@ -3,14 +3,16 @@
 
     <arg name="local" default="true"/>
 
-    <machine name="movo1" address="movo1" user="movo" 
+    <machine name="movo1" address="movo1" user="robohub" 
                  env-loader="/home/movo/env.sh" default="true" unless="$(arg local)"/>
          
     <!--System watchdog-->
     <node pkg="movo_ros" type="movo_wd" name="movo1_wd" output="screen" unless="$(arg local)">
         <param name="pc_name" value="movo1"/>
     </node>
     
+    <!-- Platform specific robot state publisher and parameters -->
```

Comment:
How do you know that this line should go into this file? Where was this file being called before?

Reply:
robot state publisher was complaining that the robot_description parameter was not being provided. the param is defined in the movo_upload.launch file